addon-chrome/actions.js

```diff
@@ -0,0 +1,296 @@
+const ChromePlacesProvider = require("addon-chrome/ChromePlacesProvider");
```

Comment:
this seems like a bit of a dumping ground of stuff - but at least it's a well documented dumping ground! so that's probably ok.